migrator-tool/src/main/scala/za/co/absa/spline/migrator/MigratorCLI.scala

```diff
@@ -35,6 +35,11 @@ object MigratorCLI extends App {
 }
 
 class MigratorCLI(migratorTool: MigratorTool) {
+
+  private val logLevels = Seq("ALL", "TRACE", "DEBUG", "INFO", "WARN", "ERROR", "OFF")
+  private val logLevelsString = logLevels.map(x => s"`$x`").reduce((x, y) => s"$x, $y")
```

Comment:
I would be scared someone will write ```--log-level `INFO` ```, would it work?

Reply:
No. The accents were already used in help message, so I reused them in error message as well. Do you think we should drop the accents from both?

Reply:
Makes sense, thx Sasa. I'd remove those back ticks.
Also, to avoid hardcoding the constants you could use something like this:
```scala
val logLevels = classOf[Level].getFields.collect { case f if f.getType == f.getDeclaringClass => f.getName }
```

Reply:
This is great.  Could this, with some genericity, fit into commons?